<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
<comment>
  * Released as open source by Gabriel Caudrelier
  *
  * Developed by Gabriel Caudrelier, gabriel dot caudrelier at gmail dot com
  *
  * https://github.com/metrodango/pip3line
  *
  * Released under BSD 3-Clause see LICENSE for more information
</comment>
 <class>IPTranslateIPv4Widget</class>
 <widget class="QWidget" name="IPTranslateIPv4Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="endianGroupBox">
     <property name="title">
      <string>Endianness</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QRadioButton" name="littleEndianRadioButton">
        <property name="text">
         <string>Little Endian</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="bigendianRadioButton">
        <property name="text">
         <string>Big Endian (Network Endian)</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="baseGroupBox">
     <property name="title">
      <string>Number base</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QRadioButton" name="hexadecimalRadioButton">
        <property name="text">
         <string>Hexadecimal (base 16)</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="decimalRadioButton">
        <property name="text">
         <string>Decimal (base 10)</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="formatGroupBox">
     <property name="title">
      <string>Hex format (only used when converting to int32 string)</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QRadioButton" name="lowerCaseRadioButton">
        <property name="text">
         <string>Lower Case</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="upperCaseRadioButton">
        <property name="text">
         <string>Upper case</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
